godel_utils/src/ensenso_guard.cpp

```diff
@@ -0,0 +1,36 @@
+#include <godel_msgs/EnsensoCommand.h>
+#include <godel_utils/ensenso_guard.h>
+
+namespace ensenso
+{
+const static std::string SERVICE_NAME = "ensenso_manager";
+
+EnsensoGuard::EnsensoGuard() : client_(nh_.serviceClient<godel_msgs::EnsensoCommand>(SERVICE_NAME))
+{
+  godel_msgs::EnsensoCommand msg;
+  msg.request.action = godel_msgs::EnsensoCommandRequest::STOP;
+  try
+  {
+    client_.call(msg);
+  }
+  catch (const std::exception e)
+  {
+    ROS_ERROR_STREAM("Exception while attempting to stop ensenso: " << e.what());
+  }
+}
+
+EnsensoGuard::~EnsensoGuard()
+{
+  godel_msgs::EnsensoCommand msg;
```

Comment:
Not a big deal because the code is small, but you could create a third helper function that takes the action to perform as an argument.

```
issueCommand(godel_msgs::EnsensoCommandRequest::STOP);
```

The constructor/destructor would just call that.